pages/authLoginDev/authLoginDev.js

```diff
@@ -0,0 +1,55 @@
+const ERR = require('async-stacktrace');
+const express = require('express');
+
+const config = require('../../lib/config');
+const csrf = require('../../lib/csrf');
+const sqldb = require('../../prairielib/lib/sql-db');
+
+var router = express.Router();
+
+router.get('/', (req, res, next) => {
+  if (!config.devMode) {
+    return next(new Error('devMode logins are not enabled'));
+  }
+
+  var authUid = config.authUid;
+  var authName = config.authName;
+  var authUin = config.authUin;
```

Comment:
Presumably this could be enhanced in the future with a form where someone could type these in on the client?

Reply:
Yeah, depending on how much of a front-end we want to put on authLoginDev, sure.